webpack.config.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * This file exists so that certain IDE's (webstorm)
+ * that can read webpack configs can resolve
+ * aliases for our folder structure.
+ * ---
+ * This is imported into next.config.js
+ */
```

Comment:
This is really great to know about -- I should set this up for the nteract monorepo similar to how we do it for flow.